quattor/types/system.pan

```diff
@@ -27,7 +27,7 @@ type structure_oldname = {
 include 'components/network/core-schema';
 
 # TODO: monitoring to be defined
-include 'monitoring/lemon/schema';
+include { if_exists('monitoring/lemon/schema') };
```

Comment:
I don't completely like the idea that `/system/monitoring` will be bound to a schema based on the existence of a template (I think this lemon template should do the `bind`) as it makes the configuration path no longer optional in the configuration. Instead I'd used a variable that must be defined (I don't remember if we can define global variables in declaration templates) and whose value must be an existing template, like `SYSTEM_MONITORING_SCHEMA`. This would allow to disable this part of the configuration on some machines even if the template exists (by not defining the variable).